packages/react-core/rollup.config.js

```diff
@@ -14,7 +13,6 @@ const config = defineConfig([
       sourcemap: false,
     },
     plugins: [
-      commonjs(),
```

Comment:
Our code base is fully written in ESM format. No longer needed

Reply:
Interesting, so we were essentially converting from CJS to ESM (required by Rollup) back to CJS?

Reply:
Correct, for our CJS source, CJS -> ESM -> CJS. But now I think we can skip the first step, i do not think we have anywhere using `require` other than the scripts and Rollup will fail since it only accepts ESM.